scripts/tests/test_swebench_lsp_toolchain.py

```
@@ -0,0 +1,2056 @@
+            with self.assertRaisesRegex(
+                TOOLCHAIN.ToolchainError, "do not share one completeness schema"
+            ):
+                TOOLCHAIN._build_frozen_cohort_manifest(cases)
+
+    def test_ready_aggregate_uses_counts_and_checkouts_without_status(self) -> None:
+        with tempfile.TemporaryDirectory() as temporary:
+            root = Path(temporary)
+            cases = []
+            report_digests = ("d" * 64, "e" * 64)
+            for index, (commit, report_digest) in enumerate(
+                zip(("a" * 40, "b" * 40), report_digests), start=1
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)